### ISSUE TITLE
https://github.com/whitequark/topkg.git does not exist

### ISSUE THREAD

**Issue by lattice0** [whitequark/opam-android#45]
https://github.com/whitequark/topkg.git does not exist anymore

**Comment by whitequark** [comment]
Ah, that's bad; I think that repo might have had changes I don't have elsewhere. Let me write to GitHub.

**Comment by whitequark** [comment]
Nope, seems like a lost cause. I think that had some contentious patches that upstream might not have liked. I unfortunately have no idea what they were, and I do not have any local copies left.

**Comment by mvglasow** [comment]
What was in that repository? Is the toolchain still usable without it, possibly with certain limitations?

**Comment by mvglasow** [comment]
The main page of the Github repo as of June 27, 2018 is archived at https://web.archive.org/web/20180627180315/https://github.com/whitequark/topkg. The last commit is `3cabbc0` (Add the --toolchain option, for ocamlbuild -toolchain.) I see upstream merged that in https://github.com/dbuenzli/topkg/pull/87. There’s also https://github.com/dbuenzli/topkg/pull/102.

Were there any other changes since then? Is the toolchain still usable without them (using upstream instead), possibly with certain limitations (if so, which)?

**Comment by whitequark** [comment]
I think that was it

**Comment by mvglasow** [comment]
So, would I just need to use the upstream repo URL instead, or omit it altogether, or...?

**Comment by whitequark** [comment]
Upstream repo, or the release should work fine
